components/survey/Card/index.jsx

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import Image from 'next/image';
+
+import forwardButton from '@/public/angle-right.svg';
+
+export const surveyCardTestIds = {
+  base: 'survey-card',
+  coverImageUrl: 'survey-card__cover_image_url',
+  title: 'survey-card__title',
+  description: 'survey-card__description',
+  forwardButton: 'survey-card__forward',
+};
+
+const SurveyCard = ({ slideId, title, description, coverImageUrl }) => (
+  <div data-test-id={surveyCardTestIds.base}>
+    <div className="w-[704px] h-[302px] mb-8" data-test-id={surveyCardTestIds.coverImageUrl}>
+      <Image className="rounded-xl" src={coverImageUrl} alt="cover image" width={704} height={302} />
+    </div>
+    <div className="flex justify-between mb-[155px]">
```

Comment:
Not sure that we should fix the bottom margin here as on some screen sizes the paginator would be pushed beyond the end of the screen 🤔 
![Screen Shot 2565-04-22 at 18 23 04](https://user-images.githubusercontent.com/1772999/164705390-70c5501b-d064-4756-a603-7f142ba8dfef.png)

Reply:
`position: fixed` for the bottom margin 😄  in 86c458775e4bb47df1b72df50f6dd54aba632aa2